utils/guidelines_xslt/odd2html/generateWebsite.xsl

```diff
@@ -1007,7 +1007,7 @@
                                 <xsl:apply-templates select=".//span[@class='specList-elementSpec']" mode="#current"/>
                             </div>
                             <div class="tile-subtitle text-gray">
-                                <xsl:apply-templates select="text()"/>
+                                <xsl:apply-templates select="text()|abbr/text()"/>
```

Comment:
Thank you @musicEnfanthen for the background! I'll work on that :)

Reply:
alternatively we could introduce a default template for all elements returning their text content? Something in the flavour of:

```xml
<xsl:template match ="*" mode="guidelines">
  <xsl:value-of select="."/>
</xsl:template>
  ```
  
  This should be at the very top of the stylesheet in order to have lower priority than any other template, so it will be only applied if the other templates don’t match.

Reply:
I have to add to that comment a bit…
As we're working in a `mode` the XSLT default templates don’t apply here so reworking those to `mode="#any"`or `mode="#default guidelines"` would be the way to go.

Reply:
The problem is/was that without a specific class in the (intermediary) HTML source the "text content" of the node() was more than just the content we needed:

```
<li class="specDesc">
    <span class="specList-elementSpec">
        <a class="link_odd link_odd_elementSpec" href="#analytic">analytic</a>
    </span>Contains bibliographic elements describing an item (<abbr>e.g.</abbr>, an article or poem) published within a monograph or journal and not as an independent publication.
</li>
```

So it always showed the "analytic" text content of the link as well. You would need something like: Give me the text content of li.specDesc, but ignore the text content coming from span.specList-elementSpec and its children. 

Introducing another global template could have side effects on other stuff we are not aware of?